### PR TITLE
Added script to check 2440

### PR DESCRIPTION
**Pull request by julsemaan**
Adds script to check inverse-inc/packetfence#2440

### CODE REVIEW

File: scripts/check-2440.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+#fname:check-2440.sh
+
+pfversion
+
+if [ "$MAINTENANCE_VERSION" = "7.0" ] || [ "$MAINTENANCE_VERSION" = "7.1" ]; then
+  if [ -f /var/lib/mysql/grastate.dat ]; then
```
Comment (dwlfrth) [inline scripts/check-2440.sh]: Shouldn't we have a "CLUSTER_IS_ENABLED" kind of function for maintenance ?
I think there are other checks that require such check

**Comment by julsemaan** [comment]
re-check that por favor

File: scripts/functions.sh [hunk 2]
```diff
@@ -103,3 +103,14 @@ github_is_reachable() {
 }
 
 export -f github_is_reachable
+
+galera_enabled() {
+  if [ -f /var/lib/mysql/grastate.dat ];then
+    return 0
```
Comment (dwlfrth) [inline scripts/functions.sh]: Hmmm, shouldn't that be 1

File: scripts/functions.sh [hunk 3]
```diff
@@ -103,3 +103,14 @@ github_is_reachable() {
...
+  else
+    return 1
```
Comment (dwlfrth) [inline scripts/functions.sh]: And here, 0 ?

Comment (julsemaan) [inline scripts/functions.sh]: No, in soviet russia bash, 0 is success, 1 is failure

Comment (julsemaan) [inline scripts/functions.sh]: Like when grep returns 0, your if succeeds, if greg returns 1, your if fails, if grep returns zammit, bash core-dumps

**Pull request merged**